src/stdune/option.mli

```diff
@@ -7,6 +7,8 @@ type 'a t = 'a option =
 module O : sig
   val (>>|) : 'a t -> ('a -> 'b  ) -> 'b t
   val (>>=) : 'a t -> ('a -> 'b t) -> 'b t
+
+  val ( let* ) : 'a t -> ('a -> 'b t) -> 'b t
```

Comment:
What about `let?`? I believe question marks is often used for operators related to options

Reply:
I don't think that's available in the syntax extension. If I try it then:

```
File "src/stdune/option.mli", line 11, characters 8-11:
Error: Syntax error: operator expected.
```

Reply:
That's sad :(, `let*` it is then...